src/main/java/randoop/contract/CompareToEqualsWithImpurity.java

```diff
@@ -0,0 +1,42 @@
+package randoop.contract;
+
+/**
```

Comment:
The comment makes it sound like Randoop should, everywhere, not perform the CompareToEquals contract on StringBuffer.  Therefore, I don't think this should be specefic to GRT Impurity fuzzing, but should be a direct change to CompareToEquals.  If you agree, could you please do that in a separate pull request?

Reply:
CompareToEquals is only done on StringBuilder when GRT-Fuzzing is on. I'll make this change now since it doesn't hurt and I can remove CompareToEqualsWithImpurity. I'll try to understand this better in the future.